### DISCUSSION TITLE
Edge Extension Store Still on 1.0.11

### DISCUSSION THREAD

**Discussion by mackenziejonesosit** [furybee/chrome-tab-modifier#408] [General]
Hey all,

Seems the Edge extension store is still on the 1.0.11 version which broke in edge v130.

Any chance we can get the new 1.0.13 version pushed to the edge store to restore function?

**Comment by sylouuu** [comment]
Hi @mackenziejonesosit,

The 1.0.13 version is currently in review in Microsoft Edge Addons.

Bests
